src/applications/vaos/appointment-list/components/cards/confirmed/VideoVisitSection.jsx

```diff
@@ -63,9 +100,37 @@ export default function VideoVisitSection({ appointment }) {
   return (
     <dl className="vads-u-margin--0">
       <dt className="vads-u-font-weight--bold">
-        How to join your virtual session
+        How to join your {isAtlas ? 'video appointment' : 'virtual session'}
       </dt>
-      <dd>{linkContent}</dd>
+      <dd>
+        {linkContent}
+        {isAtlas && (
+          <div>
+            <p>
+              <span className="vads-u-font-weight--bold">
+                Appointment Code:{' '}
```

Comment:
I know you're matching the mock, but typically we would do sentence case here: "Appointment code" instead of "Appointment Code".

Reply:
Done.